scene/gui/text_edit.cpp

```diff
@@ -4958,8 +4958,10 @@ void TextEdit::_set_line_as_hidden(int p_line, bool p_hidden) {
 
 // Symbol lookup.
 void TextEdit::_set_symbol_lookup_word(const String &p_symbol) {
-	lookup_symbol_word = p_symbol;
```

Comment:
This looks good, though would prefer it to be handed on the `CodeEdit` side in `set_symbol_lookup_word_as_valid` if possible.

Reply:
Yeah, I can move it.